A32NX/ModelBehaviorDefs/Asobo/Exterior.xml

```diff
@@ -1695,4 +1695,100 @@
+                <BLURRED_1_NODE_ID>#BLURRED_NODE_ID#</BLURRED_1_NODE_ID>
+            </DefaultTemplateParameters>
+            <OverrideTemplateParameters>
+                <PROCESS_PARAM1>True</PROCESS_PARAM1>
+                <PARAM1>NODE_ID</PARAM1>
+                <PARAM1_SUFFIX>_NODE_ID</PARAM1_SUFFIX>
+                <EXIT_TEMPLATE>A32NX_ENGINE_Turbine_Visibility_Base_Template</EXIT_TEMPLATE>
+            </OverrideTemplateParameters>
+            <UseTemplate Name="ASOBO_GT_Helper_Recursive_ID">
+                <MAX_ID>#STILL_NODE_ID_COUNT#</MAX_ID>
+                <STATE>Still</STATE>
+                <PARAM1_PREFIX>STILL_</PARAM1_PREFIX>
+            </UseTemplate>
+            <UseTemplate Name="ASOBO_GT_Helper_Recursive_ID">
+                <MAX_ID>#STILL_NODE_ID_COUNT#</MAX_ID>
+                <STATE>Slow</STATE>
+                <PARAM1_PREFIX>SLOW_</PARAM1_PREFIX>
+            </UseTemplate>
+            <UseTemplate Name="ASOBO_GT_Helper_Recursive_ID">
+                <MAX_ID>#BLURRED_NODE_ID_COUNT#</MAX_ID>
+                <STATE>Blurred</STATE>
+                <PARAM1_PREFIX>BLURRED_</PARAM1_PREFIX>
+            </UseTemplate>
+        </Component>
+    </Template>
+
+    <Template Name="A32NX_ENGINE_Turbine_Rotation_Template">
+        <UseTemplate Name="ASOBO_GT_Anim">
+            <!-- 3894 is the max rated N1 of the LEAP, times 6 is from RPM to deg/s. This seems a bit fast, so we reduce it a bit. -->
+            <ANIM_CODE>0.45 6 3894 (A:TURB ENG N1:#ID#, Percent) (A:ANIMATION DELTA TIME, seconds) * * * *</ANIM_CODE>
+            <ANIM_LENGTH>360</ANIM_LENGTH>
+            <ANIM_WRAP>1</ANIM_WRAP>
+            <ANIM_DELTA>1</ANIM_DELTA>
+        </UseTemplate>
+    </Template>
+    <Template Name="A32NX_ENGINE_Turbine_Still_Visibility_Template">
+        <UseTemplate Name="ASOBO_GT_Visibility">
+            <VISIBILITY_CODE>(A:TURB ENG N1:#ID#, Percent) #MIN_N1_PCT_FOR_SLOW# &lt;</VISIBILITY_CODE>
+        </UseTemplate>
+    </Template>
+    <Template Name="A32NX_ENGINE_Turbine_Slow_Visibility_Template">
+    <UseTemplate Name="ASOBO_GT_Visibility">
+        <VISIBILITY_CODE>
+            (A:TURB ENG N1:#ID#, Percent) #MIN_N1_PCT_FOR_SLOW# &gt;
+            (A:TURB ENG N1:#ID#, Percent) #MIN_N1_PCT_FOR_BLUR# &lt; and
+        </VISIBILITY_CODE>
+    </UseTemplate>
+    </Template>
+    <Template Name="A32NX_ENGINE_Turbine_Blurred_Visibility_Template">
+        <UseTemplate Name="ASOBO_GT_Visibility">
+            <VISIBILITY_CODE>(A:TURB ENG N1:#ID#, Percent) #MIN_N1_PCT_FOR_BLUR# &gt;</VISIBILITY_CODE>
+        </UseTemplate>
+    </Template>
+        <!-- Base visibility template for turbines -->
+    <Template Name="A32NX_ENGINE_Turbine_Visibility_Base_Template">
+        <Condition NotEmpty="NODE_ID">
+            <True>
+                <Component ID="#NODE_ID#" Node="#NODE_ID#">
+                    <UseTemplate Name="A32NX_ENGINE_Turbine_#STATE#_Visibility_Template">
```

Comment:
Here

Reply:
the state gets replaced with either "blur", "slow", or "still"

Reply:
I know, it's pretty convoluted, but that's how asobo did it, and I didn't really want to invent a whole new template